nautobot_circuit_maintenance/handle_notifications/sources.py

```diff
@@ -626,3 +648,6 @@ def get_notifications(
             )
 
     return received_notifications
+
+
+MaintenanceNotification.update_forward_refs()
```

Comment:
could you remind me why do we need to apply self referencing types? maybe I missed, but I don't see it self referencing